frontend/src/metabase/admin/tasks/containers/Help.jsx

```diff
@@ -45,12 +48,19 @@ Add any other context about the problem here.
 function githubIssueLink(bugReportDetails) {
   return (
     "https://github.com/metabase/metabase/issues/new?title=&labels=Type:Bug&body=" +
-    encodeURI(template) +
-    // replace semicolons because everything after is ignored
-    encodeURI(bugReportDetails.replace(";", " "))
+    encodeURIComponent(template) +
+    encodeURIComponent(template + "\n```json\n" + bugReportDetails + "\n```")
   );
 }
 
+function discourseLink(bugReportDetails) {
+  return (
+    "http://discourse.metabase.com/new-topic?category_id=7&body=" +
+    encodeURIComponent(template) +
```

Comment:
I don't think we want to include the bug report template here since it's a support request, just the diagnostic info.